metadata-ingestion/as-a-library.md

```diff
@@ -115,6 +115,9 @@ emitter.flush()
 ### Emitter Code
 If you're interested in looking at the Kafka emitter code, it is available [here](./src/datahub/emitter/kafka_emitter.py)
 
+## Bulk Ingestion
+The most direct way to perform ingestion via the sdk is to generate a .mcp file in the sdk and then ingest it via the [File ingestion source.](docs/generated/ingestion/sources/file.md).
```

Comment:
Probably needs a link to a how-to or a script that shows how to generate an MCP file. Also we haven't usually mandated that the extension be `.mcp`. Usually it is <>.json but we could conventionalize to `<>.mcp.json`